clients/web/apps/dashboard/src/style.css

```
@@ -1,9 +1,3 @@
-@import url("@corvus/shared/app-shell.css");
+@import url("@corvus/shared/nothing-shell.css");
```

Stylelint (17.6.0)

[error] Expected "url("@corvus/shared/nothing-shell.css")" to be ""@corvus/shared/nothing-shell.css"" (import-notation)

(import-notation)